docs/application-connector/docs/008-details-ac-security.md

```diff
@@ -15,9 +15,10 @@ You can disable the SSL certificate verification in the communication between Ky
 
 Follow these steps to disable SSL certificate verification for communication between Kyma and an existing Remote Environment:
 
-  1. Edit the `ec-default-proxy-service` Deployment in the `kyma-integration` Namespace. Run:
+  1. Edit the `{RE_NAME}-proxy-service` Deployment in the `kyma-integration` Namespace where {RE_NAME} is the name of existing Remote Environment.
```

Comment:
You don't really need to explain what `{RE_NAME}` is. It's pretty obvious on its own, and even more obvious and clear in the context of this document :)

Reply:
O.K, doc updated